test/com/google/javascript/jscomp/NewTypeInferenceES5OrLowerTest.java

```diff
@@ -11064,6 +11064,48 @@ public void testIsArrayPredicate() {
         NewTypeInference.MISTYPED_ASSIGN_RHS);
   }
 
+  public void testNumberPredicates() {
```

Comment:
Let's put these in the same method as this:
https://github.com/google/closure-compiler/blob/master/test/com/google/javascript/jscomp/NewTypeInferenceES5OrLowerTest.java#L11050